docs/guides/intro-to-graphql.md

```diff
@@ -1,9 +1,407 @@
+
+#### Usage
+
+```gql
+query {
+  User(where: { id: ID }) {
+    name
+  }
+}
+```
+
+### \_UserMeta
+
+Unlike `_allUsersMeta` this returns metadata relating to the list such as access control information. This query accepts no parameters.
+
+## Mutations
+
+For each list Keystone generates six top level mutations:
+
+- `createUser`
+- `createUsers`
+- `updateUser`
+- `updateUsers`
+- `deleteUser`
+- `deleteUsers`
+
+### createUser
+
+Add a single User to the Users list. Requires a `data` parameter that is an object where keys match the field names in the list definition and the values are the data to create.
+
+#### Usage
+
+```gql
+mutation {
+  createUser(data: { name: "Mike" }) {
+    id
+  }
+}
+```
+
+### createUsers
+
+Creates multiple users. Parameters are the same as `createUser` except the data parameter should be an array of objects.
+
+#### Usage
+
+```gql
+mutation {
+  createUsers(data: [{ name: "Mike" }]) {
+    id
+  }
+}
+```
+
+### updateUser
+
+Update a User by ID. Accepts an `id` parameter that should match the id of a user item. The object should contain keys matching the field definition of the list. Only keys that you wish to update need to be provided.
+
+#### Usage
+
+```gql
+mutation {
+  updateUser(id: ID, data: { name: "Simon" }) {
+    id
+  }
+}
+```
+
+### updateUsers
+
+Update multiple Users by ID. Accepts a single data parameter that contains an array of objects. The object parameters are the same as `createUser` and should contain an `id` and nested `data` parameter with the field data.
+
+```gql
+mutation {
+  updateUsers(data: [{ id: ID, data: { name: "Simon" } }]) {
+    id
+  }
+}
+```
+
+### deleteUser
+
+Delete a single Entity by ID. Accepts a single parameter where the `id` matches a user id.
+
+```gql
+mutation {
+  deleteUser(id: ID)
+}
+```
+
+### deleteUsers
+
+Delete multiple entities by ID. Similar to `deleteUser` where the `id` parameter is an array of ids.
+
+```gql
+mutation {
+  deleteUsers(id: [ID])
+}
+```
+
+## Executing Queries and Mutations
+
+Before you begin writing application code, a great place test queries and mutations is the [GraphQL Playground](https://www.apollographql.com/docs/apollo-server/features/graphql-playground/). The default path for Keystone's GraphQl Playground is `http://localhost:3000/admin/graphql`. Here you can execute queries and mutations against the Keystone API without writing any JavaScript.
+
+Once you have determined the correct query or mutation, you can add this to a front-end application. To do this you will need to submit a `POST` request to Keystone's API. The default API endpoint is: `http://localhost:3000/admin/api`.
+
+In our examples we're going to use the browser's [Fetch API](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API) for older browsers you may need to polyfill this or use a library like [axios](https://github.com/axios/axios).
```

Comment:
```suggestion
In our examples we're going to use the browser's [Fetch API](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API) to make a `POST` request.
```

I'm happy to leave out polyfilling. If they're supporting old browsers, they know this already, and [`fetch` has >93% support](https://caniuse.com/#search=fetch)

Reply:
Yeah I know... It's not so great on the mobile segment I think meaning still an issue for some. I'm not strong on opinions here so if you want to leave it out, fine.

Reply:
That 93% stat includes mobile too. The only ones that don't support it are mobile IE (wtf even is that? :P ) and mobile Opera (which is a horrible browser anyway). I'd prefer to remove it to avoid the cognitive pitfall of someone accidentally thinking they _need_ the polyfill.